sites/fast-website/src/public/index.html

```diff
@@ -76,7 +94,10 @@
 
         <section class="section">
           <div class="section-content">
-            Page section header
+            <fast-section-header>
+              <h5>LEARN, MODIFY AND FOLLOW</h5>
+              <h1 slot="subheading">Join the community</h1>
```

Comment:
These are definitely not h1's as an FYI. We'll want to make sure that the source is semantic and follows a logical outline.

Reply:
I think these are actually going to be something like a "badge"...I think.

Reply:
updated to be use badge